roles/ceph-osd/tasks/main.yml

```diff
@@ -49,7 +49,7 @@
   when: ceph.bcache_ssd_device is defined
 
 - name: start and add the osd service(s) to the init sequence
-  service: name=ceph
+  service: name=ceph-osd-all
```

Comment:
I think this will result in the existing `ceph` service running, and an attempt at starting the new `ceph-osd-all` service. Which is going to be weird. Also, we shouldn't be starting services before flushing handlers, otherwise we could have service start, then immediately restart due to handlers.